nirc_ehr/src/org/labkey/nirc_ehr/table/NIRC_EHRCustomizer.java

```diff
@@ -974,6 +974,20 @@ private void customizeAnimalTable(AbstractTableInfo ds)
             col.setLabel("Alias");
             ds.addColumn(col);
         }
+        if (ds.getColumn("lastProtocol") == null)
+        {
+            var col = getWrappedCol(us, ds, "lastProtocol", "demographicsLastProtocol", "Id", "Id");
+            col.setLabel("Assignment - Final Protocol");
+            col.setDescription("This calculates the final protocol assignment for the animal.  This is distinct from active protocol assignment because it will return a protocol assignment for dead animals");
+            ds.addColumn(col);
+        }
+        if (ds.getColumn("lastProject") == null)
+        {
+            var col = getWrappedCol(us, ds, "lastProject", "demographicsLastProject", "Id", "Id");
+            col.setLabel("Assignment - Final Project");
+            col.setDescription("This calculates the final project assignment for the animal.  This is distinct from active project assignment because it will return a project assignment for dead animals");
```

Comment:
Is it meant to be only for dead animals? if so, demographicsLastProject.sql should filter on dead animals?

Reply:
see comment above